src/pages/index.tsx

```diff
@@ -60,15 +60,14 @@ export default function Home(props: Props) {
               stable assets.
             </p>
           </section>
-          <CardBackground>
-            <StableTokens />
-          </CardBackground>
-          <CardBackground className="mt-[56px]">
+          <div className="flex flex-col gap-14">
```

Comment:
Why not just use the CardBackground as the div in this case?

Reply:
We use it, but just internally now. The card background isn't displayed on mobile, so we can consider the card background a part of this component and then display it conditionally from within; this main page will only lay out the components now.

Reply:
But I'm saying its redundant to have this div on line 63 when any of its styling could be applied to a card background item

Reply:
That div is the wrapping div which separates the 3 cards, there's no card background where this div should be, it's just blank and we need some spacing

Reply:
The card background which is still around stables will need to be moved as well as it's the same case where Stables doesn't need the card on mobile

Reply:
Ahh didn't see the other children, sweet